var/spack/repos/builtin/packages/julia/package.py

```diff
@@ -70,7 +70,7 @@ class Julia(Package):
     depends_on("curl")
     depends_on("git")           # I think Julia @0.5: doesn't use git any more
     depends_on("openssl")
-    depends_on("python @2.7:2.999")
+    depends_on("python @2.7.0:2.999")
```

Comment:
Spack currently selects versions such as "Python 2.7" when determining version numbers automatically. Such a version does not exist. Spack then tries to run the respective Python interpreter, and fails, because no such executable exists.

I want Spack to choose "Python 2.7.0" instead, which does exist.